dev/gpctl/cmd/clusters-update.go

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"strconv"
+
+	"github.com/spf13/cobra"
+
+	"github.com/gitpod-io/gitpod/common-go/log"
+	"github.com/gitpod-io/gitpod/ws-manager-bridge/api"
+)
+
+// clustersUpdateCmd represents the clustersListCmd command
+var clustersUpdateCmd = &cobra.Command{
+	Use:   "update [cluster name] [property] [value]",
```

Comment:
alternatively:
```
gpctl clusters update my-cluster --score=20 --max-score=100
```

Reply:
cordon and un-cordon feel like they should be separate actions/commands.
No matter that they're just properties that are set, from an operational perspective, we usually think of "to cordon an node", not to "set the cordon property of a node to X"

Reply:
> cordon and un-cordon feel like they should be separate actions/commands.

:+1: